tests/RoleTest.php

```diff
@@ -180,6 +180,14 @@ public function it_returns_false_if_it_does_not_have_a_permission_object()
         $this->assertFalse($this->testUserRole->hasPermissionTo($permission));
     }
 
+    /** @test */
+    public function it_creates_permission_object_if_it_does_not_have_a_permission_object()
+    {
+        $permission = app(Permission::class)->findOrCreate('other-permission');
+
+        $this->assertFalse($this->testUserRole->hasPermissionTo($permission));
```

Comment:
I'm not sure this is the right assertion here. Wouldn't it be better to check that the `other-permission` now exists, or that assigning `other-permission` to the user is successful?